language/predefined_spec.rb

```diff
@@ -1099,6 +1099,8 @@ def obj.foo2; yield; end
 DATA                 IO          If the main program file contains the directive __END__, then
                                  the constant DATA will be initialized so that reading from it will
                                  return lines following __END__ from the source file.
+FALSE                FalseClass  Synonym for false (deprecated in Ruby 3).
```

Comment:
```suggestion
FALSE                FalseClass  Synonym for false (deprecated, removed in Ruby 3).
```
(same for the others)